lib/cmd/fh3/app/stage.js

```diff
@@ -45,35 +45,24 @@ module.exports = {
     'gitRefValue' : i18n._("The name of the branch you'd like to deploy")
   },
   'preCmd' : function(params, cb) {
-    var data = createData(params);
-    data = setDefaultBranchAsMaster(data);
-    data = setGitRefValue(params, data);
-    data = setGitRefHash(params, data);
-
-    if (params.gitRefType) {
-      if (params.gitRefType !== 'branch' && params.gitRefType !== 'tag') {
-        return cb(i18n._('Invalid parameter for gitRefType :' + params.gitRefType));
-      }
-      data.gitRef.type=params.gitRefType;
-    }
+    if ( !params.gitRefType ) params.gitRefType = 'branch';
+    if ( !params.gitRefValue ) params.gitRefValue = 'master';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)